common/classes/com/arjuna/common/internal/util/propertyservice/BeanPopulator.java

```diff
@@ -174,6 +174,19 @@ public static void configureFromProperties(Object bean, String instanceName, Pro
         }
     }
 
+    /**
+     * Directly set a bean overriding any previous config. This method of setting a particular config
```

Comment:
This does not sound right as the bean does not override the previous state. I assume the bean is used only when there was no previous state before.

Reply:
I think it would be helpful to Narayana developers to add a note that this is not guaranteed to exist long term but as it is internal it is not essential

Reply:
I updated the javadoc to more accurately reflect what the method does. I also added Tom's caveat about the method being internal and subject to change (JBTM-3215).